src/include/daos/placement.h

```diff
@@ -69,7 +69,8 @@ struct pl_target_grp {
 struct pl_obj_shard {
 	uint32_t	po_shard;	/* shard index */
 	uint32_t	po_target;	/* target id */
-	uint32_t	po_fseq;	/* The latest failure sequence */
+	/* The latest failure sequence, see ring_obj_remap_shards(). */
```

Comment:
Actually the old comment looks better to me, this po_fseq isn't necessarily binding with ring map (we'll support mapless map soon), so 'see ring_obj_remap_shards() looks not quite appropriate to me.